src/fsspec_xrootd/xrootd.py

```diff
@@ -412,6 +412,47 @@ async def _cat_file(self, path: str, start: int, end: int, **kwargs: Any) -> Any
                 self.timeout,
             )
 
+    async def _get_file(
+        self, rpath: str, lpath: str, chunk_size: int = 8192, **kwargs: Any
```

Comment:
Any particular reason for 8192?

Reply:
No particular reason, feel free to suggest any other value.

Reply:
It seems really small to me: consider a 50MB/s download: that's 6k chunks/s. We would be using a lot of CPU just to handle the chunks with python. I would choose something more like 256kiB-1MiB

Reply:
Makes sense, I'll increase the chunk size to 256 kiB